sdk/js/src/Request.ts

```diff
@@ -67,6 +55,33 @@ export const post = async (
     })
 }
 
+export const patch = async (
+    url: string,
+    environmentKey: string,
+    body: Record<string, unknown>
+): Promise<AxiosResponse> => {
+    return await axiosClient.request({
+        method: 'PATCH',
+        url,
+        data: body,
+        headers: baseRequestHeaders(environmentKey)
+    })
+}
+
+export const getConfigJson = async (environmentKey: string, user: DVCPopulatedUser, enableCloudData?: boolean): Promise<BucketedUserConfig> => {
+    const queryParams = `${serializeUser(user)}`
+    const url = `${BASE_URL}${HOST}${CONFIG_PATH}?envKey=${environmentKey}${queryParams && '&' + queryParams}`
```

Comment:
you should construct an actual url here using the URL object, add pass queryParams around as a plain object

Reply:
don't see the need to pass around query params but i switched to making them use `URL` and `URLSearchParams`